### PR TITLE
[Web] EventKit 

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

Creates an EventKit like the BrandKit. 
  * use markdown files for content

59 change files of which
* 21 markdown files
* 9 snapshot files 
* 20 ts / tsx files 

#### new 
Screen Shot 2020-06-02 at 5 47 41 PM
Screen Shot 2020-06-02 at 5 47 57 PM
Screen Shot 2020-06-02 at 5 48 09 PM
Screen Shot 2020-06-02 at 5 48 17 PM
Screen Shot 2020-06-02 at 5 48 27 PM
Screen Shot 2020-06-02 at 5 49 07 PM
Screen Shot 2020-06-02 at 5 49 17 PM

### Related issues

- Fixes https://github.com/celo-org/celo-labs/issues/457
fixes https://github.com/celo-org/celo-labs/issues/456
fixes https://github.com/celo-org/celo-labs/issues/455
fixes https://github.com/celo-org/celo-labs/issues/460
fixes https://github.com/celo-org/celo-labs/issues/458
fixes https://github.com/celo-org/celo-labs/issues/459
fies https://github.com/celo-org/celo-monorepo/issues/3690

### Backwards compatibility

brand new

### CODE REVIEW

File: packages/web/server/EventKit.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import { Attachment, FieldSet, Table } from 'airtable'
+import airtableInit from './airtable'
+import cache from './cache'
+
+const BASE_ID = 'appjKfoHvrO5SZWdd'
```
Comment (jmrossy) [inline packages/web/server/EventKit.ts]: What's this?

File: packages/web/src/experience/Markdown.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    children,
+  ])
+
+  return (
+    <Button
+      style={standardStyles.halfElement}
+      kind={BTN.PRIMARY}
+      text={children}
+      href={href}
+      onPress={track}
```
Comment (jmrossy) [inline packages/web/src/experience/Markdown.tsx:40]: Is track the only thing that happens on press?

Comment (jmrossy) [reply packages/web/src/experience/Markdown.tsx:40]: Of is this an anchor (I see there's an href above)?

File: packages/web/src/experience/eventkit/AmbassadorPage.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import Page, { CIRCLES_PATH, ROUTE_TO_TITLE } from 'src/experience/eventkit/Page'
+import Section from 'src/experience/eventkit/Section'
+import { hashNav } from 'src/shared/menu-items'
+
+const sections = [hashNav.eventCircles.overview].map((id) => {
```
Comment (jmrossy) [inline packages/web/src/experience/eventkit/AmbassadorPage.tsx:6]: Why the [].map pattern here for a single item?

File: packages/web/src/experience/eventkit/Intro.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import Page, { ROOT, ROUTE_TO_TITLE } from 'src/experience/eventkit/Page'
+import Section from 'src/experience/eventkit/Section'
+import { hashNav } from 'src/shared/menu-items'
+
+const sections = [hashNav.eventsIntro.overview].map((id) => {
```
Comment (jmrossy) [inline packages/web/src/experience/eventkit/Intro.tsx:6]: ditto

File: packages/web/src/experience/eventkit/Tenent.tsx [hunk 5]
```diff
@@ -0,0 +1,121 @@
+import frontMatter from 'front-matter'
```
Comment (jmrossy) [inline packages/web/src/experience/eventkit/Tenent.tsx]: Nit: Should the `Tenets.tsx` (plural) if it defines them all?

File: packages/web/src/experience/eventkit/content/circles/overview.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
+# Celo Ambassador
```
Comment (jmrossy) [inline packages/web/src/experience/eventkit/content/circles/overview.md:1]: I'm skipping the markdown content, I assume experience circle will review it, or already has.

**Review by jmrossy — COMMENTED** [review]
Looks good, just a few nits.
Is this the record for most tickets closed with a single PR?

File: packages/web/server/EventKit.ts (hunk 1 above)
Comment (aaronmgdr) [reply packages/web/server/EventKit.ts]: the id for the  airtable base. I can rename for clarity

File: packages/web/src/experience/Markdown.tsx (hunk 2 above)
Comment (aaronmgdr) [reply packages/web/src/experience/Markdown.tsx:40]: yes its an anchor.

File: packages/web/src/experience/eventkit/AmbassadorPage.tsx (hunk 3 above)
Comment (aaronmgdr) [reply packages/web/src/experience/eventkit/AmbassadorPage.tsx:6]: was just less work to copy the pattern. and less work in the future when they want to add another section, (which i believe they plan on doing)

**Comment by aaronmgdr** [comment]
> Looks good, just a few nits.
> Is this the record for most tickets closed with a single PR?

possibly?

**Review by jmrossy — APPROVED** [review]

**Pull request merged**